src/FSharpFakeTargets/targets.fs

```diff
@@ -115,9 +113,15 @@ module Targets =
     run tests
   )
 
-  let private _packageTarget = _target "Package" (fun parameters ->
```

Comment:
Could we keep a `Package` target that chooses one of the two options we have now?

Might be useful to prevent breakage for consumers.